core/templates/components/question-difficulty-selector/question-difficulty-selector.component.css

```
@@ -0,0 +1,42 @@
+  display: inline;
+  font-style: italic;
+  font-weight: 400;
+  margin-right: 1.2rem;
+  width: 9rem;
+}
+
+.oppia-skill-description-container {
+  font-weight: 400;
+  margin-left: 30px;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)